ModularTegustation/harvestable_gardens.dm

```diff
@@ -0,0 +1,161 @@
+	harvest = /obj/effect/spawner/lootdrop/garden/cold
+	harvest_amount_high = 1
+	harvest_amount_low = 1
+	harvest_message_low = "You unearth a snow-covered treat."
+	harvest_message_med = "You unearth a snow-covered treat."
+	harvest_message_high = "You unearth a snow-covered treat."
+
+/obj/structure/flora/ash/garden/waste
+	name = "sickly garden"
+	desc = "Polluted water wells up from the cracked earth, feeding a patch of something curious."
+	icon_state = "gardensick"
+	harvested_name = "sickly garden"
+	harvested_desc = "Polluted water wells up from the cracked earth, where it once fed a patch of something curious. Now only wilted leaves remain."
+	harvest = /obj/effect/spawner/lootdrop/garden/sick
+	harvest_amount_high = 1
+	harvest_amount_low = 1
+	harvest_message_low = "You pry something odd from the poisoned soil."
+	harvest_message_med = "You pry something odd from the poisoned soil."
+	harvest_message_high = "You pry something odd from the poisoned soil."
+
+/obj/structure/flora/ash/garden/seaweed //yea, i code :)
+	name = "seaweed patch"
+	gender = NEUTER
+	desc = "A patch of seaweed, floating on the surface of the water"
+	icon_state = "seaweed"
+	harvested_name = "seaweed patch"
+	harvested_desc = "A patch of seaweed, floating on the surface of the water. It seems someone has already searched through this"
+	harvest = /obj/item/food/seaweed
+	harvest_amount_high = 2
+	harvest_amount_low = 1
+	harvest_message_low = "You discover some edible weeds within the patch."
+	harvest_message_med = "You discover some edible weeds within the patch."
+	harvest_message_high = "You discover some edible weeds within the patch."
+
+/obj/effect/spawner/lootdrop/garden
+	name = "lush garden seeder"
+	lootcount = 3
+	var/list/plant = list(
+			/obj/item/food/grown/citrus/lemon = 2,
+			/obj/item/food/grown/citrus/lime = 2,
+			/obj/item/food/grown/vanillapod = 2,
+			/obj/item/food/grown/cocoapod = 2,
+			/obj/item/food/grown/pineapple = 2,
+			/obj/item/food/grown/poppy/lily = 2,
+			/obj/item/food/grown/poppy/geranium = 2,
+			/obj/item/food/grown/sugarcane = 2,
+			/obj/item/food/grown/tea = 2,
+			/obj/item/food/grown/tobacco = 2,
+			/obj/item/food/grown/watermelon = 4,
+			/obj/item/grown/sunflower = 4,
+			/obj/item/food/grown/banana = 4,
+			/obj/item/food/grown/apple = 4,
+			/obj/item/food/grown/berries = 5,
+			/obj/item/food/grown/cherries = 4,
+			/obj/item/food/grown/garlic = 4,
+			/obj/item/food/grown/grapes = 4,
+			/obj/item/food/grown/grass = 5,
+			/obj/item/food/grown/pumpkin = 4,
+			/obj/item/food/grown/rainbow_flower = 4,
+			/obj/item/food/grown/wheat = 4,
+			/obj/item/food/grown/parsnip = 4,
+			/obj/item/food/grown/peas = 4,
+			/obj/item/food/grown/rice = 4,
+			/obj/item/food/grown/soybeans = 4,
+			/obj/item/food/grown/tomato = 4,
+			/obj/item/food/grown/cabbage = 4,
+			/obj/item/food/grown/onion = 4,
+			/obj/item/food/grown/carrot = 4)
```

Comment:
Large lists should end like this, github's gonna be angry if anything's added to it or removed at the end
applies to all other lists in this PR
```suggestion
			/obj/item/food/grown/carrot = 4,
		)
```